### ISSUE TITLE
Images from catbox.moe not loading. 

### ISSUE THREAD

**Issue by Imsleepyyy** [zend10/AL-chan#61]
Okay, so I was using Anitrend from the start of my animanga journey. I found Anitrend lacking so I switched to AL-chan and it was one of my best choices in life. I'm really glad I switched. So thanks for such an awesome app! 

So let's get towards the issue I want to inform. Okay, so I was writing a comment on my activity and I mainly use catbox.moe for sharing my stuff on Anilist. So I tried to do the same thing but it won't load instead it will display this <a href="https://files.catbox.moe/z1vk9h.jpg">thing. </a>

I know if I wait it will load eventually but it just won't. I surfed here and there and find out that this is happening because I'm using catbox.moe not imgur. When I tried using imgur, it worked just fine.

In short, I can't use catbox.moe, I have to use imgur for sharing my images. It's kind of inconvenient for me. Can you fix this?

**Comment by zend10** [comment]
I'm actually aware of this. Still have no idea what's going on with them but I'll probably try around with multiple image loader.
